bigquery/google/cloud/bigquery/magics.py

```diff
@@ -236,6 +237,32 @@ def use_bqstorage_api(self):
     def use_bqstorage_api(self, value):
         self._use_bqstorage_api = value
 
+    @property
+    def maximum_bytes_billed(self):
+        """int: Maximum bytes to be billed for this job or :data:`None` if not set.
+
+        Example:
+            Manually setting the context maximum_bytes_billed:
+
+            >>> from google.cloud.bigquery import magics
+            >>> magics.context.maximum_bytes_billed = '123'
+
+        Raises:
+            ValueError: If the parameters are invalid.
+        """
+        return self._maximum_bytes_billed
```

Comment:
Shouldn't this be `return job_config.maximum_bytes_billed`?

Reply:
I think this part is OK. A potential alternative implementation is to have a `self._default_job_config` object, but I think that can wait until we have more than one default property to set.

Reply:
I think we should just have a default job config that users can set properties on. That way, users can set properties on the default config that we don't have magics arguments for. No need to make properties for each of the job config properties. That approach doesn't scale well, because it would be too much repetitive code to add each job config property separately.

Reply:
I created a property for the class Context  because i found  Set job_config.maximum_bytes_billed = google.cloud.bigquery.magics.context.maximum_bytes_billed line in the description of feature request,So can i remove that property from Context class which is in magic.py file

Reply:
Sorry for the "flip flopping" @HemangChothani. Alix convinced me offline that a `default_query_job_config` property is the better implementation.

I've added commit 30f8326092e5bc16d1c4f3e9b2b1f36beedec9f8 that implements this.